libretroshare/src/pqi/authssl.cc

```diff
@@ -404,10 +404,16 @@ int AuthSSLimpl::InitAuth(
 
 		std::cout.flush() ;
 
+#ifdef RS_DIFFIE_HELLMAN_INIT_CHECK
```

Comment:
i would prefer that the define line is #ifdef SKIP_DH_INIT_CHECK, so that the default is to check when no options are enabled. Then on android only, you add the option to have this define enabled. This way, unless we explicitly want it, the check is always performed.